STATUS.md

```diff
@@ -177,6 +177,7 @@ For the current status of the low memory location implementation andd usage chec
 | `(00306)` | `$A71A`    | `QPLOP`    | NOT DONE | some implementation exists, but not connected here |
 | `(00308)` | `$A7E4`    | `GONE`     | NOT DONE | some implementation exists, but not connected here |
 | `(0030A)` | `$AE86`    | `EVAL`     | NOT DONE | some implementation exists, but not connected here |
+| `($E3A2)` |            | `MOVCHG`   | DONE     |                                                    |
```

Comment:
This suggests that $E3A2 is a vector to MOVCHG routine, which is not true; the line should be:

|         |  $E3A2 | `MOVCHG`   |

Plus, this is not an official BASIC routine - it should go to _Other unofficial BASIC routines/locations_ section.

Reply:
Ok, I'll correct this.